src/tokens/transformation/skia/templates/colors.h.template

```diff
@@ -16,3 +16,27 @@ namespace leo::dark {
   return `constexpr SkColor ${prop.name} = SkColorSetRGB(${prop.value})`;
 }).map(s => `${s};`).join('\n') %>
 } // namespace leo::dark
+
+namespace leo {
+
+enum Theme {
```

Comment:
I think `emum class` would be better as there's multiple enums in this scope.

Reply:
Sweet!